test/index.spec.js

```diff
@@ -28,7 +28,8 @@ describe('freeway', () => {
       modules: true,
       r2Buckets: bucketNames,
       // r2Persist: true
-      kvNamespaces: ['BLOCKLY']
+      kvNamespaces: ['BLOCKLY'],
+      usageModel: 'unbound'
```

Comment:
just to make test more like prod? or was it necessary for some wild reason?

Reply:
In one of the tests we read 65 indexes, and with bound usage it took us over the limit of 50.

Reply:
It also makes it more like prod ;)